### PR TITLE
Updating Lead Category to match changes we have made in Lead Central …

### PR DESCRIPTION
**Pull request by mfunkie**
…Beta so that I can migrate our code to use the Boomstrap React version

### CODE REVIEW

File: dist/Components/LeadCategory.jsx [hunk 1]
```diff
@@ -18,30 +18,54 @@ module.exports = React.createClass({
...
+      abbreviated: false,
+      small: false,
+      outline: false,
+      muted: false,
+      disabled: false
+    };
+  },
+
   render: function render() {
+    var _props = this.props;
```
Comment (igreulich) [inline dist/Components/LeadCategory.jsx]: Aren't we doing ```let``` now?

Comment (mfunkie) [inline dist/Components/LeadCategory.jsx]: This is a compiled file by babel.

Comment (igreulich) [inline dist/Components/LeadCategory.jsx]: Doh, I managed to catch that in the other PR I was looking at. (Even though you already merged it.)

**Comment by igreulich** [comment]
All in all, the PR looks good. Bump the version, and I will merge it.

**Comment by mfunkie** [comment]
K, done.

**Pull request merged**